runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -3183,7 +3185,7 @@ remoteCompile(
          statusCode = std::get<0>(recv);
          uint64_t otherData = std::get<1>(recv);
          if (statusCode == compilationLowPhysicalMemory && otherData != -1) // if failed due to low memory, should've received an updated memory state
-            updateCompThreadActivationPolicy(compInfoPT, (JITServer::ServerMemoryState) otherData); 
+            updateCompThreadActivationPolicy(compInfoPT, (JITServer::ServerMemoryState) otherData, (JITServer::ServerActiveThreadState) otherData); 
```

Comment:
I am not a fan of lying here. We receive a `JITServer::ServerMemoryState` and pretend it's a `JITServer::ServerActiveThreadState`. Currently, the two data structures have the same number of values with more or less the same meaning, but in the future that might change.
It's better to use a safe value for the second parameter `JITServer::ServerActiveThreadState::NORMAL_THREAD` so that the decisions to suspend are based only on the first parameter.